website/project/model.py

```diff
@@ -1364,6 +1365,14 @@ def save(self, *args, **kwargs):
 
         saved_fields = super(Node, self).save(*args, **kwargs)
 
+        ultimate_parent = self._primary_key
+        if getattr(self, 'parent', None):
+            # TODO Remove if statement after ultimate_parent migration
```

Comment:
It's possible to avoid a follow-up PR by interacting with the database directly, but if you would rather `DO` this afterwards to avoid the headache, I have no objections.